grep/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM komand/python-3-slim-plugin:2
+FROM komand/python-3-37-slim-plugin
```

Let's pin this to version 3
```suggestion
FROM komand/python-3-37-slim-plugin:3
```